### PR TITLE
Implemented login feature & added Splash screen for iOS.

### PR DESCRIPTION
**Pull request by joaquinTop**
## FEATURE: LOGIN
---
**NOTE:** This is my first pr, i know there are a lot of files included but most of them are related to the fact that i used [this](https://www.npmjs.com/package/react-native-rename) package which handles renaming a lot of files.

### PR include the following:
* Splash screen ( only for iOS )
* Login
* Logout

### Screenshot:
screen shot 2017-10-18 at 11 37 39 am

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,43 +1,49 @@
...
-	},
-	"jest": {
-		"preset": "react-native"
-	}
+  "name": "TargetMVD",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "start": "node node_modules/react-native/local-cli/cli.js start",
+    "lint": "eslint src/**",
```
Comment (fedeagripa) [inline package.json:7]: Adding another script with watch option could help you during development. But is up to you to decide if it's actually useful or not

File: src/actions/userActions.js [hunk 2]
```diff
@@ -21,10 +21,10 @@ export const login = user =>
   (dispatch) => {
     dispatch(submitLogin());
     return userApi.login({ user })
-    .then(({ user }) => {
+    .then((user) => {
       sessionService.saveUser(user)
       .then(() => dispatch(loginSuccess()));
-    }).catch(() => {
+    }).catch((err) => {
```
Comment (fedeagripa) [inline src/actions/userActions.js]: are you using this err?

**Review by fedeagripa — APPROVED** [review]
Great work!!! Just a few comments :)

File: src/actions/userActions.js (hunk 2 above)
Comment (joaquinTop) [reply src/actions/userActions.js]: Done, ```err``` was unused.

**Pull request merged**